app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] It looks like your sign-up and login button does not work as shown on the video 👇🏻 by the way my console shows that the user has been successfully created and login however I'm still on the same page. Please address this 👍🏻  

[Screencast from 10-14-2022 01:17:27 AM.webm](https://user-images.githubusercontent.com/79058364/195728004-0f000586-331a-47e5-ac22-53f202f1236d.webm)